### DISCUSSION TITLE
动态过滤器怎么添加关键词？

### DISCUSSION THREAD

**Discussion by lsh1215225** [the1812/Bilibili-Evolved#2040] [提问 / Q&A] [answered]
没找到添加入口

**Comment by the1812** [comment, marked as answer]
<img width="300" src="https://user-images.githubusercontent.com/26504152/125399829-b34a3b80-e3e3-11eb-8bee-102af01b4ca4.png">

**Reply by lsh1215225** [reply]
![image](https://user-images.githubusercontent.com/20274559/125400747-daedd380-e3e4-11eb-901e-d7dcaea72690.png)
我怎么没这个呢

**Reply by the1812** [reply]
在动态首页

**Reply by lsh1215225** [reply]
找到了 感谢
